modules/openy_features/openy_prgf/modules/openy_prgf_gallery/openy_prgf_gallery.module

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * @file
+ * OpenY Paragraph Gallery module file.
+ */
+
+/**
+ * Implements hook_preprocess_node().
+ */
+function openy_prgf_gallery_preprocess_node(&$variables) {
+  // Attach openy_prgf_gallery_alert.js.
+  $variables['#attached']['library'][] = 'openy_prgf_gallery/gallery_alert';
```

Comment:
What if node doesn't have this paragraph at all? This function will attache library to every single node. Let's find a bettew way of attaching libraries.